Added 404 error page

arpit3018 opened the pull request:
1. Changed learn/templates/404.html file

mohitkh7 commented on learn/templates/404.html:
```diff
@@ -1,4 +1,619 @@
+                for whatever is missing or take a look around the rest of our site.</p>
+
+            <div id="navigation">
+                <a class="navigation" href="{% url 'index' %}">Home</a><a class="navigation" href="">Go Back</a>
```
Go back URL is not working as expected.
It should go to previous valid URL, but it is reloading the same page again.

arpit3018 replied:
I have'nt added any href to go back url

arpit3018 replied:
Added Go back URL.

mohitkh7 commented:
There should be a navbar on the top. Most of the websites 404 page have a navbar having atleast website name and logo.

mohitkh7 commented on learn/templates/404.html:
```diff
@@ -1,4 +1,619 @@
+            }
+        }
+    </style>
+    <title>Start Learning</title>
```
Page title should be something more relevant like `Easy Learning - 404 Page Not Found`

arpit3018 replied:
Okay

arpit3018 commented:
1. Added Go Back URL
2. Added Logo and Name at the left top corner of the page
3. Added hyperlink to the logo and the name 
4. Create a separate file for js and css and removed iniline css and js.
